app/uk/gov/hmrc/economiccrimelevyreturns/controllers/ReturnSubmittedController.scala

```diff
@@ -51,7 +51,7 @@ class ReturnSubmittedController @Inject() (
     val amountDue: BigDecimal = BigDecimal(
       request.session
         .get(SessionKeys.AmountDue)
-        .getOrElse(throw new IllegalStateException("Amount due not found in session"))
+        .getOrElse(throw new IllegalStateException("Amount due not found in return data"))
```

Comment:
`Amount due not found in session` was fine, as this is pulling data from the session :)

Reply:
sure - difference here is that you are pulling data from the session, in the check your answers controller you are pulling data from mongo - so they are distinctive 😃